packages/cli/src/__tests__/commands/Version.test.ts

```diff
@@ -60,10 +60,14 @@ describe('version', () => {
 
   // Binary Tests
 
-  testIf(!useNodeAPI)('basic version', async () => {
-    const data = await ctx.cli('--version')
-    expect(cleanSnapshot(data.stdout)).toMatchSnapshot()
-  })
+  testIf(!useNodeAPI)(
+    'basic version',
+    async () => {
+      const data = await ctx.cli('--version')
+      expect(cleanSnapshot(data.stdout)).toMatchSnapshot()
+    },
+    50000,
+  )
```

Comment:
I had to increment the Jest timeout, otherwise buildkite would complain (not Github Actions, though).

Reply:
Huh, that is a bit concerning. Is it really that slow or slower than before?

Reply:
In practice, no, it's not slower. Buildkite seems to think otherwise 🤷 

It may be related to [this other comment](https://github.com/prisma/prisma/pull/14760#issuecomment-1216312847).